Uzoamaka126/My-Auto-Mart-Challenge
# #165906956 Implement signup interface

#### What does this PR do?
Implement signup interface

#### Description of Task to be completed?
- [x] Design and implementation of AutoMart Signup interface

#### How should this be manually tested?
* Clone https://github.com/Uzoamaka126/My-Auto-Mart-Challenge
* Visit `/signup.html` on your browser
* You should see the image below

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
#165906956

#### Screenshots (if appropriate)
![Screenshot (193)](https://user-images.githubusercontent.com/46677256/57585280-28f1ae80-74de-11e9-9a89-c8da5aa85542.png)

#### Questions:
None
